### DISCUSSION TITLE
ModuleNotFoundError:No module named 'models'

### DISCUSSION THREAD

**Discussion by ArshiyaDev** [pytest-dev/pytest#11136] [Q&A]
project/
├── docs/
├── src/
│   ├── models/
│   │   └── domain/
│   │       └── businesses.py
│   └── tests/
│       └── units/
│           └── test_businessesmodel.py
└── ...

Hello when cd to tests/units after that run pytest test_businessesmodel.py i got error no module anemd models i add __init__.py file to tests and units but again i get this error can somebody help me ?

**Comment by nicoddemus** [comment]
Hi,

This is a problem with your setup, make sure to follow [Good Integration Practices](https://docs.pytest.org/en/stable/explanation/goodpractices.html) such as installing your project with `pip` and use a `pyproject.toml`.
